lib/resources/styles.css

```diff
@@ -72,6 +72,7 @@ main {
 ::-webkit-scrollbar-thumb{ background-color: #CCC; }
 ::-webkit-scrollbar-thumb:hover{ background-color: #CCC; }
 ::-webkit-scrollbar{ width: 4px; }
+.main-content::-webkit-scrollbar{ width: 8px; }
```

Comment:
lgtm, but what do things look like if you just remove all the explicit `webkit-scrollbar` selectors?